logistics/scripts/modules/packaging/nuget-helper.psm1

```diff
@@ -3,6 +3,9 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
+& "$PSScriptRoot\..\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"
+Import-Module -Force -Scope Global (Get-RepositoryResolvedPath "logistics\scripts\modules\utility\cross-platform.psm1")
```

Comment:
Let's change those to forward slashes rather than propagating more backslashes...

Reply:
I realized while testing on Unix that this actually didn't cause issue...do we still want to do it anyway?